packages/formation/.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    [
+      "babel-preset-env",
+      {
+        "targets": {
+          "node": "current"
```

Comment:
You should copy the settings from vets-website's babelrc, I think.

Reply:
Sure

Reply:
Probably not the React ones, but I think this would also be used for the formation.js output, so it would need to have the same environment targets and ES2015+ proposal plugins.

Reply:
Okay, sound good.